test/e2e/network/ingressclass.go

```diff
@@ -73,7 +72,7 @@ var _ = common.SIGDescribe("IngressClass [Feature:Ingress]", func() {
 		}
 	})
 
-	ginkgo.It("should prevent Ingress creation if more than 1 IngressClass marked as default [Serial]", func() {
+	ginkgo.It("should choose the one with the lower name if more than 1 IngressClass marked as default [Serial]", func() {
```

Comment:
I'm confused, I think that the criteria is to choose the one with lower creation timestamp, if both timestamp are equal, the lower name

Reply:
> I'm confused, I think that the criteria is to choose the one with lower creation timestamp, if both timestamp are equal, the lower name

Actually, it's the one with the later creation timestamp.And in this test case, they all have the same timestamp. I have updated the text to disambiguate.